roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -46,18 +46,25 @@ def run(self, tmp=None, task_vars=None):
 
         result["checks"] = check_results = {}
 
+        user_disabled_checks = [
+            check.strip()
+            for check in task_vars.get("openshift_disable_check", "").split(",")
```

Comment:
I make a brief mention of this var in https://github.com/openshift/openshift-docs/pull/4452